### ISSUE TITLE
Explain what the optimal line length for content is

### ISSUE THREAD

**Issue by timpaul** [alphagov/govuk-design-system#1126]
A user was asking in the accessibility email group about optimal line length for content. We don't appear to give any recommendations about this.

We should recommend that paragraphs are no wider than 2/3rds the width of a standard page on GOV.UK (assuming you're using the default paragraph font size). This guidance could go with the 'paragraph' section, or perhaps the page layout section too.

**Comment by nickcolley** [comment]
Related: https://github.com/alphagov/govuk-frontend/issues/1656

I believe that most guidance I've seen on this refers to the amount of characters (60ish) rather than the size of a page. It's also worth mentioning that on smaller viewports 2/3rds width is full width.

**Comment by nickcolley** [comment]
> ## Short sentences
> People with some learning disabilities read letter for letter - they do not bounce around like other users. They also cannot fully understand a sentence if it’s too long.
> 
> People with moderate learning disabilities can understand sentences of 5 to 8 words without difficulty. By using common words we can help all users understand sentences of around 25 words.

https://www.gov.uk/guidance/content-design/writing-for-gov-uk#short-sentences

> ## Success Criterion 1.4.8 Visual Presentation (Level AAA)
> For the visual presentation of blocks of text, a mechanism is available to achieve the following:
> 
> - Foreground and background colors can be selected by the user.
> - **Width is no more than 80 characters or glyphs (40 if CJK).**
> - Text is not justified (aligned to both the left and the right margins).
> - Line spacing (leading) is at least space-and-a-half within paragraphs, and paragraph spacing is at least 1.5 times larger than the line spacing.
> - Text can be resized without assistive technology up to 200 percent in a way that does not require the user to scroll horizontally to read a line of text on a full-screen window.

https://www.w3.org/TR/WCAG21/#visual-presentation (note this is level AAA accessibility regulations require only level AA)

**Comment by nickcolley** [comment]
- https://insidegovuk.blog.gov.uk/2014/08/04/sentence-length-why-25-words-is-our-limit/
- https://designnotes.blog.gov.uk/2015/09/16/tips-for-creating-good-typography/

**Comment by StephenGill** [comment]
Added some text to explain this: https://github.com/alphagov/govuk-design-system/pull/1152
